task.sql

```diff
@@ -1 +1,7 @@
--- write your code for database user creation here
\ No newline at end of file
+-- write your code for database user creation here
+
+CREATE USER 'webappuser'@'%' IDENTIFIED BY 'P@ssw0rd';
+GRANT SELECT, INSERT, UPDATE, DELETE ON ShopDB.* TO 'webappuser'@'%';
+
+CREATE USER 'deploymentuser'@'%' IDENTIFIED BY 'P@ssw0rd';
+GRANT ALL PRIVILEGES ON ShopDB.* TO 'deploymentuser'@'%';
```

Comment:
Add code that apply changes and fix EOF

Reply:
Can you please explain which changes I should apply?

Reply:
I mean command FLUSH PRIVILEGES